api/src/main/java/me/lokka30/treasury/api/economy/currency/CurrencyManager.java

```diff
@@ -65,6 +65,102 @@ public void registerCurrency(@NotNull Currency currency) {
         }
     }
 
+    /**
+     * @param currencies currencies to register
+     * @see #registerCurrency(Currency)
+     */
+    public void registerCurrency(@NotNull Currency @NotNull ... currencies) {
+        Objects.requireNonNull(currencies, "currencies");
+        for (Currency currency : currencies) {
+            registerCurrency(currency);
+        }
+    }
+
+    /**
+     * @param currencies currencies to register
+     * @see #registerCurrency(Currency)
+     */
+    public void registerCurrency(@NotNull Iterable<@NotNull Currency> currencies) {
+        Objects.requireNonNull(currencies, "currencies");
+        for (Currency currency : currencies) {
+            registerCurrency(currency);
+        }
+    }
+
+    /**
+     * Unregisters the specified {@link Currency} {@code currency} if it has been registered.
+     *
+     * @param currency the currency you want unregistered
+     * @author MrIvanPlays
+     * @since v1.0.0
+     */
+    public void unregisterCurrency(@NotNull Currency currency) {
+        Objects.requireNonNull(currency, "currency");
+        BY_ID.remove(currency.getCurrencyId());
+        for (String acceptableName : currency.getCurrencyNames()) {
+            BY_NAME.remove(acceptableName);
+        }
+        if (currency.getCurrencyCharacter() != null) {
+            BY_CHAR.remove(currency.getCurrencyCharacter());
+        }
+    }
+
+    /**
+     * @param currencies the currencies you want unregistered
+     * @see #unregisterCurrency(Currency)
+     */
+    public void unregisterCurrency(@NotNull Currency @NotNull ... currencies) {
```

Comment:
Should this be `unregisterCurrencies`?

Reply:
why should it be

Reply:
Because it allows to unregister several currencies at ones.
I do prefer the name we have now